addons/mrp_subcontracting/tests/test_subcontracting.py

```diff
@@ -621,6 +621,84 @@ def test_mrp_report_bom_structure_subcontracting(self):
         self.assertEqual(report_values['lines']['components'][0]['bom_cost'], 25)
         self.assertEqual(report_values['lines']['components'][1]['bom_cost'], 5)
 
+    def test_mrp_report_bom_structure_subcontracting_quantities(self):
```

Comment:
I don't know if you remember but I had issue with this test failing when run locally.
It still the case (still the same error, `AssertionError: 80.0 != 100`), if you install `mrp_subcontracting` only, this test will fail.
You need `purchase_mrp` to make it pass, I don't know why yet